src/main/php/text/json/WrappedFormat.class.php

```diff
@@ -70,11 +71,25 @@ protected function formatObject($value) {
   }
 
   public function open($token) {
-    return $token."\n".str_repeat($this->indent, $this->level++);
+    if ('{' === $token || empty($this->stack)) {
+      $this->stack[]= $this->comma;
+      $indent= str_repeat($this->indent, $this->level++);
+      $this->comma= ",\n".$indent;
+      return $token."\n".$indent;
+    } else {
+      $this->stack[]= $this->comma;
+      $this->comma= ', ';
+      return $token;
+    }
   }
 
   public function close($token) {
-    $this->level--;
-    return "\n".str_repeat($this->indent, $this->level - 1).$token;
+    $this->comma= array_pop($this->stack);
+    if ('}' === $token || empty($this->stack)) {
```

Comment:
The `empty($this->stack)` here prevents us from having to change a test's expected outcome *but* is an inconsistency.

Reply:
However, it's consistent with the top-level doc comment which states: *Wrapped JSON format - indents objects and first-level arrays.*